src/libraries/System.Diagnostics.Process/src/System/Diagnostics/AsyncStreamReader.cs

```diff
@@ -88,6 +88,8 @@ internal void CancelOperation()
         // This is the async callback function. Only one thread could/should call this.
         private async Task ReadBufferAsync()
         {
+            await Task.CompletedTask.ConfigureAwait(ConfigureAwaitOptions.ForceYielding);
```

Comment:
Is this Task.Yield but skipping the sync context?

Reply:
Yes